### ISSUE TITLE
clip==1.0 import error

### ISSUE THREAD

**Issue by Hasnat79** [jaehong31/SAFREE#1]
Hi I'm getting the following error while trying to install via requirements.txt

ERROR: Could not find a version that satisfies the requirement clip==1.0 (from versions: 0.0.1, 0.1.0, 0.2.0)
ERROR: No matching distribution found for clip==1.0

instead, I'm using ```pip install git+https://github.com/openai/CLIP.git```  
Let me know if openai version of clip will be okay for this project

**Comment by jaehong31** [comment]
Yes, `pip install git+https://github.com/openai/CLIP.git` should work!
